ext/xsl/php_xsl.stub.php

```diff
@@ -6,49 +6,49 @@ class XSLTProcessor
 {
     /**
      * @param DOMDocument|SimpleXMLElement $stylesheet
-     * @return bool
+     * @tentative-return-type
      */
-    public function importStylesheet(object $stylesheet) {}
+    public function importStylesheet(object $stylesheet): bool {}
 
     /**
      * @param DOMDocument|SimpleXMLElement $document
-     * @return DOMDocument|false
+     * @tentative-return-type
      */
-    public function transformToDoc(object $document, ?string $returnClass = null) {}
+    public function transformToDoc(object $document, ?string $returnClass = null): DOMDocument|false {}
 
     /**
      * @param DOMDocument|SimpleXMLElement $document
-     * @return int
+     * @tentative-return-type
      */
-    public function transformToUri(object $document, string $uri) {}
+    public function transformToUri(object $document, string $uri): int {}
 
     /**
      * @param DOMDocument|SimpleXMLElement $document
-     * @return string|false|null
+     * @tentative-return-type
      */
-    public function transformToXml(object $document) {}
+    public function transformToXml(object $document): string|false{}
```

Comment:
By looking at the implementation, I'm not convinced that `null` is a possible return type.